src/api/todos.ts

```diff
@@ -0,0 +1,16 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const USER_ID = 271;
+
+export const getTodos = () => {
+  return client.get<Todo[]>(`/todos?userId=${USER_ID}`);
+};
+
+export const createTodo = (newTodo: Omit<Todo, 'id'>) => {
+  return client.post<Todo>(`/todos?userId=${USER_ID}`, newTodo);
+};
+
+export const deleteTodo = (todoId: number) => {
+  return client.delete(`/todos?userId=${todoId}`);
```

Comment:
wrong delete endpoint
```suggestion
  return client.delete(`/todos/${todoId}`);
```

![chrome-capture-2024-3-16 (1)](https://github.com/mate-academy/react_todo-app-add-and-delete/assets/33029864/2949b997-9951-47a5-949f-a8768c188665)